lib/transmuxer/ts_transmuxer.js

```diff
@@ -187,14 +188,45 @@ shaka.transmuxer.TsTransmuxer = class {
    */
   transmux(data, stream, reference, duration, contentType) {
     const ContentType = shaka.util.ManifestParserUtils.ContentType;
+
+    const uint8ArrayData = shaka.util.BufferUtils.toUint8(data);
+
+    let transmuxData;
+    if (shaka.util.TsParser.probe(uint8ArrayData)) {
+      transmuxData = this.tsTransmuxing_(
+          uint8ArrayData, stream, reference, duration, contentType);
+    } else if (contentType == ContentType.AUDIO) {
+      transmuxData = this.aacTransmuxing_(
```

Comment:
So we just assume that any audio content that isn't TS is AAC?

Is that safe?

Would we ever expect to see someone do the same with MP3 as was done in this bug with AAC?

Reply:
I changed the entire implementation to reduce duplicate code.